pkg/common/config/types.go

```diff
@@ -163,6 +163,10 @@ type GCConfig struct {
 	TanzuKubernetesClusterName string `gcfg:"tanzukubernetescluster-name"`
 	// Cluster Distribution Name
 	ClusterDistribution string `gcfg:"cluster-distribution"`
+	// GCAPIVersion refers to the API version of the object guest cluster is created from.
+	GCAPIVersion string `gcfg:"gc-api-version"`
```

Comment:
I think we agreed on the name right? 
We should use `ClusterAPIVersion` and `ClusterKind`.

Reply:
Okay I will change it.

Reply:
Done